test/helper_test.py

```diff
@@ -38,6 +38,30 @@
 EVAL_STR='if (resource_info["width"] < 450) { upload_options["tags"] = "a,b" }; ' \
          'upload_options["context"] = "width=" + resource_info["width"]'
 
+ADDON_ALL = 'all'                                                   # Test all addons.
```

Comment:
Please extract to a separate file

Reply:
Done in https://github.com/cloudinary/pycloudinary/pull/265